test/hotspot/gtest/utilities/test_byteswap.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
As a new file this should only have 2023 copyright year - thanks.

Reply:
Fixed.